pkg/roll/run_update.go

```diff
@@ -455,7 +476,14 @@ func rollAlgorithm(old, new, desired, minHealthy int) (int, int) {
 	// remaining>0 and headroom>0. each one imposes an upper bound on how
 	// many we can schedule at once, so return the minimum of the two
```

Comment:
err yeah irrelevant was a strong word but it seems to suggest there is only one return value